src/App.js

```diff
@@ -21,7 +21,7 @@ import SignUp from "./pages/SignUp";
 import Contact from "./pages/Contact";
 import Cart from "./pages/cart";
 import Wishlist from "./pages/wishList";
-import ComparePage from "./pages/compare";
```

Comment:
`import ComparePage from "./pages/compare";`

Need this component for 
Line 275: `<Route exact={true} path="/compare" element={<ComparePage />} />`